etc/spack/defaults/windows/packages.yaml

```
@@ -20,3 +20,6 @@ packages:
       cxx: [msvc]
       mpi: [msmpi]
       gl: [wgl]
+  mpi:
+    require:
+    - one_of: [msmpi]
```

(Suggestion, not a request) This shorter syntax should also work:
```suggestion
    - msmpi
```